test/Microsoft.Azure.Devices.Edge.Test.TempSensor/Test.cs

```diff
@@ -139,6 +148,7 @@ public class Args
             public string HubImage;
             public string SensorImage;
             public Option<(string address, string username, string password)> Registry;
+            public TimeSpan Timeout;
```

Comment:
nit: should we rename as "TestTimeout" to avoid confusion?

Reply:
`Args` is nested inside the `Test` class. `Test.Args.TestTimeout` seems redundant.